ImprovedAnvil/lib/spells/03_patches_vanilla.tph

```diff
@@ -75,6 +75,172 @@ COPY_EXISTING ~SPIN999.spl~ ~override/SPIN999.spl~
 
 // class level abilities //
 
+// Seeking Sword
```

Comment:
code readability here (and in similar situations) is better if there were new lines delimiting logical blocks (like an extra new line after each patch_for_each .. end section)

Reply:
Fixed